LCM/scripts/InstallModule.py

```diff
@@ -285,7 +286,21 @@ def main(args):
 
         resourceLibraryFileDestinationPath = join(helperlib.CONFIG_LIBDIR, resourceLibraryFileDestinationName)
 
+        # Python 2.4 and 3 recognize different formats for octal
```

Comment:
Can you put the oct variable assignment/conversion before the loop so that the script doesn't have to assign the variable for each resource?

Reply:
Done.